yarn.lock

```diff
@@ -10627,17 +10628,17 @@ __metadata:
   linkType: hard
 
 "@wordpress/router@npm:^1.8.0, @wordpress/router@npm:^1.8.1":
-  version: 1.8.1
-  resolution: "@wordpress/router@npm:1.8.1"
+  version: 1.11.0
```

Comment:
These changes look weird. Could you reset the file to the trunk branch and run `yarn` again?